oh.ps1

```diff
@@ -810,211 +795,263 @@ cd "$OH_PATH" # workaround for hard coded paths
+		"v"	{ # show version
+	        	Write-Host "--------- Software version ---------"
+			Get-Content $OH_PATH\$OH_DIR\rsc\version.properties | Where-Object {$_.length -gt 0} | Where-Object {!$_.StartsWith("#")} | ForEach-Object {
+			$var = $_.Split('=',2).Trim()
+			New-Variable -Force -Scope Private -Name $var[0] -Value $var[1] 
+			}
+			Write-Host "Open Hospital version" $VER_MAJOR $VER_MINOR $VER_RELEASE
+			Write-Host "MySQL version: $MYSQL_DIR"
+			Write-Host "JAVA version: $JAVA_DISTRO"
+			Write-Host ""
+			# show configuration
+	 		Write-Host "--------- Script Configuration ---------"
+	 		Write-Host "Architecture is $ARCH"
+	 		Write-Host "Config file generation is set to $GENERATE_CONFIG_FILES"
+			Write-Host ""
+	 		Write-Host "--------- OH Configuration ---------"
+	 		Write-Host "Open Hospital is configured in $OH_MODE mode"
+			Write-Host "Language is set to $OH_LANGUAGE"
+			Write-Host "Demo data is set to $DEMO_DATA"
+			Write-Host "Log level is set to $LOG_LEVEL"
+			Write-Host ""
+			Write-Host "--- Database ---"
+			Write-Host "DATABASE_SERVER=$DATABASE_SERVER"
+			Write-Host "DATABASE_PORT=$DATABASE_PORT"
+			Write-Host "DATABASE_NAME=$DATABASE_NAME"
+			Write-Host "DATABASE_USER=$DATABASE_USER"
+			Write-Host ""
+			Write-Host "--- Dicom ---"
+			Write-Host "DICOM_MAX_SIZE=$DICOM_MAX_SIZE"
+			Write-Host "DICOM_STORAGE=$DICOM_STORAGE"
+			Write-Host "DICOM_DIR=$DICOM_DIR"
+			Write-Host ""
+			Write-Host "--- OH Folders ---"
+			Write-Host "OH_DIR=$OH_DIR"
+			Write-Host "OH_DOC_DIR=$OH_DOC_DIR"
+			Write-Host "OH_SINGLE_USER=$OH_SINGLE_USER"
+			Write-Host "CONF_DIR=$CONF_DIR"
+			Write-Host "DATA_DIR=$DATA_DIR"
+			Write-Host "BACKUP_DIR=$BACKUP_DIR"
+			Write-Host "LOG_DIR=$LOG_DIR"
+			Write-Host "SQL_DIR=$SQL_DIR"
+			Write-Host "SQL_EXTRA_DIR=$SQL_EXTRA_DIR"
+			Write-Host "TMP_DIR=$TMP_DIR"
+			Write-Host ""
+			Write-Host "--- Logging ---"
+			Write-Host "LOG_FILE=$LOG_FILE"
+			Write-Host "LOG_FILE_ERR=$LOG_FILE_ERR"
+			Write-Host "OH_LOG_FILE=$OH_LOG_FILE"
+			Write-Host ""
+
+			Read-Host;
+		}
+		###################################################
+		"X"	{ # clean
+			Write-Host "Cleaning Open Hospital installation..."
+			clean_files;
+			clean_database;
+			Write-Host "Done!"
+			Read-Host;
+		}
+		###################################################
+		"q" 	{ # quit
+			Write-Host "Quit pressed. Exiting.";
+			exit 0; 
+		}
+		###################################################
+		"Q"	{ # Quit
+			Write-Host "Quit pressed. Exiting.";
+			exit 0; 
+		}
+		###################################################
+		""	{ # Start
+			Write-Host "Starting Open Hospital...";
+			$opt="Z";
+		}
+		###################################################
+		default { Write-Host "Invalid option: $opt."; 
+			Read-Host;
+			break;
+		}
+		}
+	Clear-Host;
 	}
+	# execute until quit is pressed or CLIENT/PORTABLE/SERVER mode is select (Z option)
+	until ( ($opt -ceq 'q') -Or ($opt -ceq 'Z') )
```

Comment:
above in lines 1031 and 1036 the test is for both lower case and upper case Q; the *until* only tests for lowercase; shouldn't it be case insensitive?  (my shell scripting is rusty)

Reply:
It's not you but the awful poweshell syntax, you are right !
Powershell the Case-Sensitive Comparison (-ceq) -> Correctet with -ieq, thank you !